src/ui/pages/equipment/asset/AssetGeneral.js

```diff
@@ -9,7 +9,7 @@ import { isDepartmentReadOnly, isMultiOrg } from 'ui/pages/EntityTools';
 import EAMUDF from 'ui/components/userdefinedfields/EAMUDF';
 import GridWS from 'eam-components/dist/ui/components/eamgrid/lib/GridWS';
 
-async function fetchHasHazards(equipmentCode) {
+async function fetchSafetyData(equipmentCode) {
```

Comment:
Since you are doing all the logic inside `StatusRow.js` why do you create the function here to then resolve the promise on the other component. You could move this function to Status Row and call the await as you invoke the method.

Reply:
I was thinking the same thing but since you specifically told me to pass the data to the child component, I thought that in a component perspective it would make maybe more sense to fetch here. But I'll move it.